Makefile

```diff
@@ -18,7 +18,8 @@ format:
 
 .PHONY: test_e2e
 test_e2e:
-	pytest -vv -n 5 ${PYTEST_FLAGS} tests/e2e
+	pytest -vv -n 5 ${PYTEST_FLAGS} tests/e2e -m "not serial"
+	pytest -vv -n 0 ${PYTEST_FLAGS} tests/e2e -m "serial"
```

Comment:
note: since `neuro login` is global per-user, tests on transfer can't run in parallel with any other test.